src/modules/feature-modules/admin/services/users-validation-rules.service.ts

```diff
@@ -0,0 +1,132 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { map, take } from 'rxjs/operators';
+
+import { CoreService } from '@app/base';
+
+import { UrlModel } from '@app/base/models';
+
+
+type AdminOperationType = 'LOCK_USER' | 'UPDATE_USER_ROLE' | 'CHANGE_UNIT';
+type AdminOperationRuleType = 'AssessmentUserIsNotTheOnlyOne' | 'LastAccessorUserOnOrganisationUnit' | 'LastAccessorFromUnitProvidingSupport';
+
+export type AdminValidationResponseDTO = {
+  validations: {
+    rule: AdminOperationRuleType,
+    valid: boolean,
+    data?: {
```

Comment:
No attr for the obj?

Reply:
This will be revisited when change role and change unit stories come into place... ;)